spec.html

```diff
@@ -30460,12 +30460,7 @@ <h1>Object ( [ _value_ ] )</h1>
 
     <emu-clause id="sec-properties-of-the-object-constructor">
       <h1>Properties of the Object Constructor</h1>
-      <p>The Object constructor:</p>
-      <ul>
-        <li>has a [[Prototype]] internal slot whose value is %Function.prototype%.</li>
-        <li>has a *"length"* property whose value is *1*<sub>𝔽</sub>.</li>
-        <li>has the following additional properties:</li>
-      </ul>
+      <p>The Object constructor has a [[Prototype]] internal slot whose value is %Function.prototype%. It also has the following properties:</p>
```

Comment:
There are multiple sections which still uses the "ul" listing even with 2 items
  * https://tc39.es/ecma262/#sec-properties-of-the-boolean-constructor
  * https://tc39.es/ecma262/#sec-properties-of-the-symbol-constructor
  * https://tc39.es/ecma262/#sec-properties-of-the-symbol-constructor
  * and some more

and also I don't see any existing paragraph that uses the new style.

So I think it's better keeping the "ul" listing style, with the length bullet removed, for the consistency.

Reply:
> I don't see the benefit of this kind of consistency.

One benefit is easier automated processing + checking.

E.g., the "WebIDL in ES" effort wants "to produce a representation of ECMAScript built-ins that could be automatically generated from the ECMAScript specification HTML file", and these `<ul>` elements appear to be relevant to that.